app/presenters/manuscript_metadata_presenter.rb

```diff
@@ -58,12 +58,20 @@ def render?
 
     def fields
       document_show_fields(document).select do |_, field|
-        field.section == type && should_render_show_field?(document, field)
+        field_is_correct_type?(field) && should_render_show_field?(document, field)
       end
     end
 
     private
 
+    # Force the ehxibit_tags field to be grouped in the :general section even though it is
+    # not configured in the controller and therefore can't be grouped via our normal means
+    def field_is_correct_type?(field)
```

Comment:
Not married to this solution at all.  Happy to explore other options if folks have good ideas.

Reply:
I can't remember how this all works; I wonder if a `before_action` in the catalog controller could set the right section?